content/en/docs/tasks/configure-pod-container/security-context.md

```diff
@@ -419,6 +419,43 @@ securityContext:
     localhostProfile: my-profiles/profile-allow.json
 ```
 
+## Set the AppArmor Profile for a Container
+
+To set the AppArmor profile for a Container, include the `appArmorProfile` field
+in the `securityContext` section of your Pod or Container manifest. The
+`appArmorProfile` field is a
+[AppArmorProfile](/docs/reference/generated/kubernetes-api/{{< param "version"
+>}}/#apparmorprofile-v1-core) object consisting of `type` and `localhostProfile`.
+Valid options for `type` include `RuntimeDefault`, `Unconfined`, and
+`Localhost`. `localhostProfile` must only be set if `type: Localhost`. It
```

Comment:
One interesting aspect of setting the AppArmor: if `RuntimeDefault` is specified explicitly, than Pod will not be admitted if AppArmor is not enabled on the Node. However if not specified and the default (which is also `RuntimeDefault`) will be used, than Pod can be scheduled when AppArmor is disabled.

This semantical difference is important for 3rd party tooling that needs to work universally on any cluster. Worth mentioning

Reply:
Thanks! Added a note highlighting this.